content/en/getting_started/opentelemetry/_index.md

```diff
@@ -0,0 +1,283 @@
+            enabled: true
+          container.network.io.usage.tx_packets:
+            enabled: true
+          container.cpu.usage.system:
+            enabled: true
+          container.memory.rss:
+            enabled: true
+          container.blockio.io_serviced_recursive:
+            enabled: true
+          container.uptime:
+            enabled: true
+          container.memory.hierarchical_memory_limit:
+            enabled: true
+    ```
+
+2. Update `service.pipelines.metrics.receivers` to include `docker_stats`:
+
+    ```yaml
+    service:
+      pipelines:
+        metrics:
+          receivers: [otlp, docker_stats] # <- update this line!
+    ```
+
+This configuration allows the Calendar application to send container metrics to Datadog for you to explore in Datadog.
+
+### Using unified tagging to correlate observability data
+
+[Unified service tagging][6] ties observability data together in Datadog so you can navigate across metrics, traces, and logs with consistent tags.
+
+The Calendar application is already configured with unified service tagging:
+
+1. Go to the Calendar application's Docker Compose file at `./deploys/docker/docker-compose-otel.yml`.
+2. The following lines enable the correlation between application traces and other observability data: 
+
+    ```yaml
+    environment:
+      - OTEL_SERVICE_NAME=calendar-otel
+      - OTEL_LOGS_EXPORTER=otlp
+      - OTEL_RESOURCE_ATTRIBUTES=deployment.environment=docker,host.name=otelcol-docker
+    ```
+
+
+## Running the application to generate observability data
+
+All of the initial setup and configuration is complete. To start generating and forwarding observability data to Datadog, you need to run the Calendar application with the OpenTelemetry SDK:
+
+1. Run the application from the `calendar/` folder:
+   ```sh
+   docker compose -f deploys/docker/docker-compose-otel.yml up
+   ```
+
+   This command creates a Docker container with the OpenTelemetry Collector and the Calendar service.
+
+2. To test that the Calendar application is running correctly, execute the following command from another terminal window:
+   ```sh
+   curl localhost:9090/calendar 
+   ```
+
+3. Verify that you receive a response like:
+   ```
+   {“date":"2022-12-30"}
+   ```
+
+Each call to the Calendar application results in metrics, traces, and logs being forwarded to the OpenTelemetry Collector, then to the Datadog Exporter, and finally to the Datadog backend. 
+
+**Note**: The Calendar application uses the probabilistic sampler processor, so only 30% of traces sent through the application reach the target backend. Run the `curl` command several times to ensure at least one trace exports to the Datadog backend. 
+
+## Exploring observability data in Datadog
+
+Now, you can use Datadog to explore the Calendar application's observability data.   
+
+**Note**: It may take a few minutes for your trace data to appear in Datadog.
+
+### Exploring runtime and infrastructure metrics
```

Comment:
It would be great to also add a section to illustrate infra host list view for the workflow of exploring Infrastructure level health in this section.

Reply:
So you are thinking we need to expand the Infrastructure Metrics note into a section that also shows what specifically is shown in there (e.g., CPU Usage by Pod)?